org-code-javabuilder/media/src/main/java/org/code/media/Image.java

```diff
@@ -76,6 +71,10 @@ public Image(int width, int height) {
    * @return Pixel at the given coordinate
    */
   public Pixel getPixel(int x, int y) {
+    if (this.pixels == null) {
+      this.createPixelArray();
+      this.bufferedImage = null;
```

Comment:
we don't explicitly need to set bufferedImage to null, but since we are running on a fairly constrained set of resources I'm setting it to null to indicate that it can be garbage collected.